src/Nethermind/Nethermind.Blockchain/Filters/Topics/AnyTopic.cs

```diff
@@ -21,12 +21,16 @@ namespace Nethermind.Blockchain.Filters.Topics
 {
     public class AnyTopic : TopicExpression
     {
-        public static readonly AnyTopic Instance = new(); 
+        public static readonly AnyTopic Instance = new();
+
+        private AnyTopic() { }
         
         public override bool Accepts(Keccak topic) => true;
         public override bool Accepts(ref KeccakStructRef topic) => true;
 
         public override bool Matches(Bloom bloom) => true;
         public override bool Matches(ref BloomStructRef bloom) => true;
+        
+        public override string ToString() => "null";
```

Comment:
It looks strange a bit. What is the reason?

Reply:
In request AnyTopic is passed as null, so when we are ToStringing it we want same representation. SO it comes from JSON RPC spec.